guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend.md

```diff
@@ -35,7 +35,7 @@ Although Magento architecture favors loosely coupled software components, module
 
 * libraries (either Magento Framework {% glossarytooltip 08968dbb-2eeb-45c7-ae95-ffca228a7575 %}library{% endglossarytooltip %} or third party libraries)
 
-<div class="bs-callout bs-callout-warning" id="warning">
+<div class="bs-callout bs-callout-warning" markdown="1">
```

Comment:
This is called out as a warning, but in the actual content it says "Note:". This seems incorrect.

Reply:
This issue is out of scope in this PR.